<launch>

  <!-- BEGIN ROBOT 1-->
  <!-- <group ns="robot1"> -->
  <!-- <arg name="name" />
  <group ns="$(arg name)"> -->
    <node name="landmark_location_service_server" pkg="tm_motion" type="landmark_location_service_server.py" output="screen" />
    <node name="move_action_server" pkg="tm_motion" type="move_action_server.py" output="screen" />
      <!-- <param name="ip_address" value="172.21.5.123" />
      <param name="port" value="7171" /> -->
      <arg name="ip" />
      <param name="ip_address_tm" value="$(arg ip)" />
      <!-- <param name="port" value="5890" /> -->
  <!-- </group> -->


</launch>
